velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -73,6 +73,12 @@ class Re2FunctionsTest : public test::FunctionBaseTest {
       const std::vector<std::optional<T>>& groupIds,
       const std::vector<std::optional<std::vector<std::string>>>& output);
 
+  void testRe2ExtractAll(
+      std::string source,
```

Comment:
pass strings as const &

What is idx? Is this a group number? Would you add comments for this method?